### ISSUE TITLE
Mid-2013 Macbook Air

### ISSUE THREAD

**Issue by hadro** [nodanaonlyzuul/crystal-ruby-digest-shootout#2]
Macbook Air Mid-2013 13inch)
10.11.6 (15G1108)
1.7 GHz Intel Core i7
8gb

Crystal: 
```
./shootout  43.91s user 5.76s system 90% cpu 54.952 total
```

Ruby:
```
real 0m29.260s
user 0m22.306s
sys 0m4.834s
```

Let me know if you need more?

**Comment by nodanaonlyzuul** [comment]
Thanks @hadro , Can you format the Crystal results like you did for the Ruby results?
I plan on sending these issues to the crystal-lang committers.
You're doing the lord's work.

**Comment by hadro** [comment]
Yeah, for some reason my computer is spitting them out with different formatting for the two, but lemme figure it out, one moment
